ldk/javascript/src/webpack/config.ts

```diff
@@ -1,96 +1,22 @@
 import * as webpack from 'webpack';
 import * as path from 'path';
-import Terser from 'terser-webpack-plugin';
-import { generateBanner } from './generate-banner';
-import { LdkSettings } from './ldk-settings'; // Need to dynamically refer to Loop's package.json
+import { LdkSettings } from './ldk-settings';
+import { buildBabelConfig, buildOptimization, buildWebpackConfig } from './shared';
 
+// Need to dynamically refer to Loop's package.json
```

Comment:
Rich, is this a TODO?

Reply:
Looks like he is telling us why he disabled the linter, as he needs to use a dynamic require to get the loop's package.json

Reply:
What Michelle said :)